packages/plugin-betswirl/src/actions/coinToss.ts

```diff
@@ -0,0 +1,233 @@
+import { z } from "zod";
+import { type Hex } from "viem";
+import {
+    type Action,
+    type IAgentRuntime,
+    type Memory,
+    type HandlerCallback,
+    type State,
+    elizaLogger,
+    type ActionExample,
+    composeContext,
+    generateObject,
+    ModelClass,
+} from "@moxie-protocol/core";
+import { MoxieWalletClient } from "@moxie-protocol/moxie-agent-lib/src/wallet";
+import {
+    CASINO_GAME_TYPE,
+    CoinToss,
+    COINTOSS_FACE,
+    slugById,
+    formatTxnUrl,
+} from "@betswirl/sdk-core";
+import { casinoBetParams, getMaxBetCountParam } from "../types";
+import {
+    getChainIdFromWallet,
+    getBetToken,
+    placeBet,
+    getBet,
+    getBetAmountInWei,
+} from "../utils/betswirl";
+import { formatTokenForMoxieTerminal } from "../utils/moxie";
+
+export const CoinTossBetParameters = z.object({
+    face: z.nativeEnum(COINTOSS_FACE).describe("The face of the coin"),
```

Comment:
Make this nullable so it can have `null` value without throwing any validation error

```suggestion
    face: z.nativeEnum(COINTOSS_FACE).describe("The face of the coin").nullable(),
```